install.sh

```diff
@@ -188,15 +188,15 @@ check_deps() {
 }
 
 check_python() {
-  if which python2.7 &> /dev/null; then
+  if type -P python2.7 &> /dev/null; then
```

Comment:
Good catch. We probably should use `has` defined above, which calls `type` as well.